rest-api-spec/build.gradle

```diff
@@ -83,6 +83,7 @@ tasks.named("yamlRestTestV7CompatTransform").configure { task ->
   task.skipTest("search/370_profile/fetch source", "profile output has changed")
   task.skipTest("search/370_profile/fetch nested source", "profile output has changed")
   task.skipTest("search/240_date_nanos/doc value fields are working as expected across date and date_nanos fields", "Fetching docvalues field multiple times is no longer allowed")
+  task.skipTest("search/110_field_collapsing/field collapsing and rescore", "#107779 Field collapsing is compatible with rescore in 8.15")
```

Comment:
Do you still need to skip when the test is deleted?

Reply:
It seems so yes, these tests are sourced from the previous version afaik.